recipes/zlib-ng/all/conanfile.py

```diff
@@ -0,0 +1,114 @@
+from conans import ConanFile, CMake
+from conans.tools import download, unzip, replace_in_file, remove_files_by_mask, get
+import os
+
+class ZlibNgConan(ConanFile):
+    name = "zlib-ng"
+    description = "zlib data compression library for the next generation systems"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage="https://github.com/Dead2/zlib-ng/"
+    license="https://github.com/Dead2/zlib-ng/blob/ba7f0eb6e294306ac6d771216ea4442f2ea2d830/LICENSE.md"
```

Comment:
If I'm not mistaken, this is the ZLIB license.
See https://spdx.org/licenses/Zlib.html
```suggestion
    license ="Zlib"
```